ServerCore/Pages/Index.cshtml

```diff
@@ -21,14 +21,21 @@
         </td>
         <td></td>
     </tr>
-</table><br />
-<p>A one-weekend event for FTEs and friends that happens every summer. Puzzle solvers of all levels of experience are welcome!</p>
-<a href="https://puzzlehunt.azurewebsites.net/nipd2021/play">Go to Non-Intern Puzzleday 2021</a>
+</table>
+<br />
+<p>A two-day event over one week for interns and friends that's happening for the first time this spring. Puzzle solvers of all levels of experience are welcome!</p>
```

Comment:
Is there a reason to state that this event has a primarily international focus?

Reply:
Yeah, probably a good call to be a little more explicit about why we're doing this since it's the first time.  Will make the update in the next commit!